### PR TITLE
Local storage

### PR DESCRIPTION
**Pull request by Isabellanamutenda**
In this project I

- Updated my README file
- Fix my linters errors
- Add javascript code for local storage
- I have fixexd the error and this is the output
- This is the last chance I hope I did everythin right
- before
Screenshot from 2021-08-06 18-48-12
after
Screenshot from 2021-08-06 19-45-09

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -132,11 +132,27 @@ const form = document.getElementById('contact-form');
...
   if (email.value !== email.value.toLowerCase()) {
     msg.style.display = 'block';
     e.preventDefault();
   }
+  const data = {
+    name: inputName.value,
+    email: email.value,
+    message: inputMessage.value,
+  };
+  localStorage.setItem('data', JSON.stringify(data));
```
Comment (oxenprogrammer) [inline script.js:148]: **Change Required**
- [ ] _After filling the form, and not submitting it, the entered values should be saved in the browser._ The current behavior only saves the data in the browser after form submit. Please add this functionality.

**Review by oxenprogrammer — CHANGES_REQUESTED** [review]
Hi **Isabella,**

Good job so far!
I can see you're really putting effort to become a world-class software developer :+1:
There are some issues that you still need to address in order to move forward to the next project.
Trust me you got this! :muscle:


## Required Changes ♻️
- [x] The _README_ file contains `typos`  and doesn't explicitly direct the person interested in the project on the specific set of command to perform to get the project up and running. Remember the _README_ acts as a documentation which allows other interested parties to know exactly how to collaborate or understand your project.

_Check the comments under the review. for Project Specific Requirements_

## Optional suggestions
_Consider the optional changes nice to have features and only work on them when you have satisfied the required changes_

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: script.js [hunk 2]
```diff
@@ -132,11 +132,27 @@ const form = document.getElementById('contact-form');
...
 }
 
 form.addEventListener('submit', validationEmail);
+
+window.addEventListener('load', () => {
+  const data1 = JSON.parse(localStorage.getItem('data')) || [];
+  inputName.value = data1.name;
+  email.value = data1.email;
+  inputMessage.value = data1.message;
+});
```
[the remark this answers is not in the document]
Comment (Isabellanamutenda) [reply script.js]: Resolved

File: script.js (hunk 1 above)
Comment (Isabellanamutenda) [reply script.js:148]: Resolved

Comment (abruzy) [reply script.js:148]: Hi @Isabellanamutenda ,

This issue is yet to be resolved, when I fill the form without submitting and refreshing the page, it doesn't save it, but the other behaviour works fine

**Review by abruzy — CHANGES_REQUESTED** [review]
Hi @Isabellanamutenda 👋🏽, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Lameck1 — APPROVED** [review]
Hi, @Isabellanamutenda, @TahenyBELGUITH 

You've done a great job implementing the previously requested changes. Your project is complete!  It's time to merge it :shipit: 

Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Pull request merged**